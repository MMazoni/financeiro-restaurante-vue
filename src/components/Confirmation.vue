<template>
  <v-row justify="center">
    <v-dialog v-model="dialog" max-width="290">
      <v-card>
        <v-card-title class="headline">Você tem certeza?</v-card-title>

        <v-card-actions>
          <v-spacer></v-spacer>

          <v-btn color="secondary darken-1" text @click="runConfirmation(false)">Não</v-btn>

          <v-btn color="secondary darken-1" text @click="runConfirmation(true)">Sim</v-btn>
        </v-card-actions>
      </v-card>
    </v-dialog>
  </v-row>
</template>

<script>
import { confirmation } from "../store.js";

export default {
  name: "Confirmation",
  computed: {
    dialog() {
      return confirmation.dialog;
    }
  },
  methods: {
    runConfirmation(enable) {
      confirmation.dialog = enable;
      confirmation.confirm = enable;
    }
  },
};
</script>
