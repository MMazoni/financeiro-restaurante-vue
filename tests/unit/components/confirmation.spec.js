import { mount } from '@vue/test-utils';
import Confirmation from '@/components/Confirmation.vue';


describe('Confirmation.vue', () => {

  it('should snapshot', () => {
    const wrapper = mount(Confirmation);
    expect(wrapper).toMatchSnapshot();
  });

  it('is a Vue instance', () => {
    const wrapper = mount(Confirmation);
    expect(wrapper.vm).toBeTruthy();
  });
});

