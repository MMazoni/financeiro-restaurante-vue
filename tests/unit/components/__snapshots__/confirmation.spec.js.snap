// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmation.vue should snapshot 1`] = `
<div class="row justify-center">
  <div role="dialog" class="v-dialog__container">
    <!---->
  </div>
</div>
`;
